cmake/OpenCVFindLibsPerf.cmake

```diff
@@ -84,6 +84,12 @@ if(WITH_EIGEN AND NOT HAVE_EIGEN)
         set(EIGEN_WORLD_VERSION ${EIGEN3_WORLD_VERSION})
         set(EIGEN_MAJOR_VERSION ${EIGEN3_MAJOR_VERSION})
         set(EIGEN_MINOR_VERSION ${EIGEN3_MINOR_VERSION})
+      elseif(DEFINED Eigen3_VERSION_MAJOR)
+        # see https://github.com/opencv/opencv/issues/27530
+        # Case sensitive is needed to support Eigen on Master branch at 13 July,2025.
```

Comment:
It'll be great to add pointer to Eigen commit or MR instead of just date.

Reply:
Thank you for your comment, I update comment to point Eigen commit.

Eigen master drop to deprecated packge config variables EIGEN3_VERSION_*.

https://gitlab.com/libeigen/eigen/-/commit/f2984cd0778dd0a1d7e74216d826eaff2bc6bfab

Currently status are...
- On Eigen master, we can use only Eigen3_VERISION_*. 
- On Eigen 3.4 branch, we can use both Eigen3_VERSION_* and EIGEN3_VERSION_* .